TaskMessage[] Tauc
---
TaskMessage bid
float64 time
string robot_name